### PR TITLE
# Improve UX. Added lambda code size

### PR DESCRIPTION
**Pull request by CCristi**

### CODE REVIEW

**Comment by CCristi** [comment]
http://deep.dev.public.7638c213.s3-website-us-east-1.amazonaws.com/#/deep-benchmarking

File: src/DeepBenchmarking/Backend/resources.json [hunk 1]
```diff
@@ -48,5 +48,15 @@
       ],
       "source": "src/CaseWithoutFramework"
     }
+  },
+  "retrieve": {
+    "lambda-size": {
```
Comment (mgoria) [inline src/DeepBenchmarking/Backend/resources.json]: Please use `retrieve` as action name and `lambda-size` as resource name

File: src/DeepBenchmarking/Backend/resources.json [hunk 2]
```diff
@@ -48,5 +48,15 @@
...
+      "description": "testing lambda performance",
```
Comment (mgoria) [inline src/DeepBenchmarking/Backend/resources.json]: Update description

File: src/DeepBenchmarking/Backend/src/RetrieveLambdaSize/Handler.es6 [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        FunctionList: Joi.array().includes(Joi.string().required()).required()
+      });
+    }
+  }
+
+  /**
+   * @returns {Number}
+   */
+  get REQUEST_INTERVAL() {
+    return 200;
```
Comment (mgoria) [inline src/DeepBenchmarking/Backend/src/RetrieveLambdaSize/Handler.es6]: Try to use a smaller interval or increase lambda execution time from 1min -> 5min

File: src/DeepBenchmarking/Frontend/js/app/angular/controllers/DeepBenchmarkingMainController.js [hunk 4]
```diff
@@ -177,12 +179,17 @@ export class DeepBenchmarkingMainController {
...
         if (index < loops) {
           setTimeout(function() {
-            execRequest(index);
+            execRequest(lambdaSize, index);
           }, intervalMs);
         }
-    };
+    }
+
+    this._lambdaSizePromise.then((sizeMap) => {
```
Comment (mgoria) [inline src/DeepBenchmarking/Frontend/js/app/angular/controllers/DeepBenchmarkingMainController.js:186]: Add error handler

**Pull request merged**